e2e/app/component/annotations-sidebar.ts

```diff
@@ -72,22 +71,22 @@ export default class AnnotationsSidebar extends BaseHelpSidebar {
 
   // click on the Annotations EDIT button to edit the annotations field name
   async getAnnotationsEditModal(): Promise<EditDeleteAnnotationsModal> {
-    const button = await Button.findByName(this.page, {name: LinkText.Edit}, this);
+    const button = Button.findByName(this.page, { name: LinkText.Edit }, this);
     await button.click();
     const modal = new EditDeleteAnnotationsModal(this.page);
     await modal.waitForLoad();
     return modal;
   }
 
   // text area
-  async getAnnotationsTextArea(): Promise<Textarea> {
```

Comment:
i'm not 100% familiar with typescript yet but i would prefer if we kept these `async` and `await` keywords unless they really aren't needed for some reason? i looked up the rule that's doing this and if these really aren't async then i'm fine with deleting them

Reply:
Yes, these functions don't have async calls inside, so I was able to remove `await` modifier. This type of error was easy to fix. But affected many files.
Two eslint rules which led the change:
`no-return-await`
`require-await`